Aleksandr Zorin/index.js

```diff
@@ -0,0 +1,111 @@
+function splitAndMerge(str, sp) {
+    var words = str.split(' ');
+    var newWords = [];
+    words.forEach(function (word) {
+        newWords.push(word.split('').join(sp))
+    });
+    return newWords.join(' ');
```

Comment:
@Zorin-Alex Works good. Lok at `split` method deeper, it can take regexp as an argument, so you can split all the string with `split(/\s|/))` and then join in one string